### PR TITLE
add cvt rpm to csm pit image

### PR DESCRIPTION
**Pull request by jyothsna-j-deshpande**
### Summary and Scope

- Fixes: CASMCVT-177
- Requires:
- Relates to:

#### Issue Type

- Bugfix Pull Request
- Docs Pull Request
- RFE Pull Request

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [ ] I tested this on internal system (if yes, please include results or a description of the test)
- [ ] I tested this on a vagrant system (if yes, please include results or a description of the test)
- [ ] I tested this on a vshasta system (if yes, please include results or a description of the test)

### CODE REVIEW

**Comment by heemstra** [comment]
@jyothsna-j-deshpande - please include the associated Jira in the PR title.

File: repos/cray.template.repos [hunk 1]
```diff
@@ -11,3 +11,6 @@ https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifac
 
 # SDU
 https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/dst-rpm-mirror/sdu-rpm-stable-local/release/img_oci-shasta-2.1/sle15_sp3_ncn?auth=basic                                   sdu-rpm-stable-local                              --no-gpgcheck -p 89                  cray/sdu/sle-15-sp3
+
+# CVT
+https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/dst-rpm-mirror/csm-diags-rpm-stable-local/release/csm-diags-1.4.0/sle15_sp3?auth=basic                                    csm-diags-sle-15sp3-1.4.0                  --no-gpgcheck -p 89                   cray/cvt/sle-15sp3
```
Comment (rustydb) [inline repos/cray.template.repos:16]: I heard an SP4 version was coming, assuming that's coming soon please update this link before merging.

**Comment by jyothsna-j-deshpande** [comment]
> @jyothsna-j-deshpande - please include the associated Jira in the PR title.

Added it

File: repos/cray.template.repos (hunk 1 above)
Comment (jyothsna-j-deshpande) [reply repos/cray.template.repos]: Addressed

**Comment by rustydb** [comment]
Moved to: https://github.com/Cray-HPE/metal-provision/pull/90

**Pull request closed without merging**